package.json

```diff
@@ -128,10 +128,10 @@
             "<rootDir>/scripts/jest/setup.ts"
         ],
         "moduleNameMapper": {
-            "^@root(.*)$": "<rootDir>$1",
-            "^@frontend(.*)$": "<rootDir>/packages/frontend$1",
             "^svgs/(.*)$": "<rootDir>/__mocks__/svgMock.tsx",
-            "^(.*)\\.svg$": "<rootDir>/__mocks__/svgMock.tsx"
+            "^(.*)\\.svg$": "<rootDir>/__mocks__/svgMock.tsx",
```

Comment:
This is an ordering issue - now that the svgs live in the frontend package, we need to match them first to mock them in the tests.